README.md

```diff
@@ -29,12 +29,62 @@ tcld version
 ```
 
 # Authentication and Login
+### User login authentication:
 In order to use the cli you must first login by running the following command:
 ```
 tcld login
 ```
 You will be sent a link to confirm your device code and login. After logging in, you are now authenticated and can make requests with this cli.
 
+### API Key based authentication:
+You can use API keys to authenticate with the cli by passing the `--api-key` flag or setting the `TEMPORAL_CLOUD_API_KEY` environment variable.
+```
+tcld --api-key <api-key> ...
+```
+
+```
+export TEMPORAL_CLOUD_API_KEY=<api-key>
+tcld ...
+```
+
+# API Key Management (Preview)
+<em>The API Key feature is currently in "Preview Release". Customers must be invited to use this feature. Please reach out to Temporal Cloud support for more information.</em>
```

Comment:
Alternatively `_` or `*` is the markdown syntax, just happens to display fine in GitHub since it supports HTML tags.